### DISCUSSION TITLE
[@types/react-datepicker] react-datepicker v5 and v6

### DISCUSSION THREAD

**Discussion by tyteen4a03** [DefinitelyTyped/DefinitelyTyped#68485] [Issues with a @types package]
Hi, is it possible to get a package update for @types/react-datepicker for react-datepicker@5 and @6?

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
See #68437 for `react-datepicker@6` types

**Comment by tyteen4a03** [comment]
Thanks for releasing v6.0.1 - can we get v6.1.0 released as well?
